scripts/device-steps.sh

```diff
@@ -20,10 +20,9 @@ AGENTS="$AGENTS0 $AGENTS1"
 
 PATH=$BINDIR:$PATH
 
-STARTTIME=`date`
-echo "Starting device-steps.sh at" $STARTTIME
-echo "go-provision version:" `cat $BINDIR/versioninfo`
-echo "go-provision version.1:" `cat $BINDIR/versioninfo.1`
+echo $(date -Ins -u) "Starting device-steps.sh"
```

Comment:
Does this work? `date -Ins -u` is not valid in `busybox` (on which `alpine` is based) or BSD-based distros.

Reply:
It works fine for me on the device inside the container.
But we have
lrwxrwxrwx 1 root root 20 Apr 17 02:33 /bin/date -> ../usr/bin/coreutils

Reply:
Ah ok. As long as it works there it is good.